wscript

```diff
@@ -1170,13 +1170,15 @@ def qemu_image_spi(ctx):
 
         # Pad with 0xFF up to image size
         tail_padding_size = image_size - resources_begin - len(res_img)
-        qemu_spi_img_file.write("\xff" * tail_padding_size)
-
-    with open(os.devnull, 'w') as null:
-        qemu_spi_cooker_node = ctx.path.get_bld().make_node('qemu_spi_cooker')
-        qemu_spi_cooker_path = qemu_spi_cooker_node.path_from(ctx.path)
-        subprocess.check_call([qemu_spi_cooker_path, spi_flash_path], stdout=null)
+        qemu_spi_img_file.write(bytes([0xff]) * tail_padding_size)
 
+        # qemu_spi_cooker is broken on OSX but it doesn't really matter
```

Comment:
nit: open an issue to track this

Reply:
Created https://github.com/pebble-dev/pebble-firmware/issues/30